api/models/HDescription.js

```diff
@@ -38,7 +38,8 @@ module.exports = {
     },
 
     dateReviewed: {
-      type: 'ref',
+      type: 'string',
```

Comment:
why switching from ref to string if columnType: 'timestamp' ?

Reply:
We did that because we use this attribute as primary key and it must be string or number. We will check if there is other possibility.